packages/bpk-component-accordion/stories.js

```diff
@@ -18,304 +18,31 @@
-    </BpkAccordion>
-  ))
-  .add('Custom', () => (
-    <BpkAccordion>
-      <BpkAccordionItem id="stops" title="Stops" tagName="h3" expanded>
-        <StopsContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem
-        id="departure-times"
-        title="Departure times"
-        tagName="h2"
-      >
-        <AirlinesContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem id="journey-duration" title="Journey duration" expanded>
-        <AirportsContent />
-      </BpkAccordionItem>
-    </BpkAccordion>
-  ))
-  .add('Custom title textStyle', () => (
-    <SingleItemAccordion>
-      <BpkAccordionItem
-        textStyle="lg"
-        id="stops"
-        title="Stops"
-        initiallyExpanded
-      >
-        <StopsContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem textStyle="lg" id="airlines" title="Airlines">
-        <AirlinesContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem textStyle="lg" id="airports" title="Airports">
-        <AirportsContent />
-      </BpkAccordionItem>
-    </SingleItemAccordion>
-  ))
-  .add('With icons', () => (
-    <BpkAccordion>
-      <BpkAccordionItem
-        id="stops"
-        title="Stops"
-        tagName="h3"
-        expanded
-        icon={<AlignedStopsIcon fill={colorPanjin} />}
-      >
-        <StopsContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem
-        id="departure-times"
-        title="Departure times"
-        tagName="h2"
-        icon={<AlignedTimeIcon fill={colorMonteverde} />}
-      >
-        <AirlinesContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem id="journey-duration" title="Journey duration" expanded>
-        <AirportsContent />
-      </BpkAccordionItem>
-    </BpkAccordion>
-  ))
-  .add('With bold titles', () => (
-    <SingleItemAccordion>
-      <BpkAccordionItem
-        id="stops"
-        title="Stops"
-        initiallyExpanded
-        weight={WEIGHT_STYLES.bold}
-      >
-        <StopsContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem
-        id="airlines"
-        title="Airlines"
-        textStyle="lg"
-        weight={WEIGHT_STYLES.bold}
-      >
-        <AirlinesContent />
-      </BpkAccordionItem>
-      <BpkAccordionItem
-        id="airports"
-        title="Airports"
-        textStyle="xl"
-        weight={WEIGHT_STYLES.bold}
-      >
-        <AirportsContent />
-      </BpkAccordionItem>
-    </SingleItemAccordion>
-  ));
+  .add('Single item only', SingleItemExample)
+  .add(
+    'Single item only (second item initially expanded)',
+    SingleItemExampleInitiallyExpanded,
+  )
+  .add('Multiple items open', MultipleItemsOpenExample)
+  .add(
+    'Multiple items open (second & third items initially expanded)',
+    MultipleItemsOpenInitiallyExpandedExample,
+  )
+  .add('Custom', CustomExample)
+  .add('Custom title textStyle', CustomTitleTextStyleExample)
+  .add('With icons', WithIconsExample)
+  .add('With bold titles', WithBoldTitlesExample);
```

Comment:
I wonder if we could use a helper that does something similar to CSF where we give it a list of components and it automatically creates the stories from them.
In other words we give it `SingleItemOnly` and it infers the story name `Single item only`.

Maybe yagni?

Reply:
Yeah I was wondering about something like this too actually. The advantage would be that we could build something to automatically create the stories based on whatever is in `examples.js` so we wouldn't even need to have all these `stories.js` files.

It's a nice idea but it'd be a lot more work so I reckon _for now_ we call it out of scope and we might come back to it later?